### ISSUE TITLE
[help] My Chia GUI connecting to Others Wallet?

### ISSUE THREAD

**Issue by byekity** [Chia-Network/chia-blockchain#3120]
Under connections type I see that I am connected to wallets' of strangers with different IPs. Right after opening 8444 port. I began to see "wallet type" connections along with Full Node type. I didn't see such connections except my wallet before. Is this normal if port is open? My wallet and funds are safe? If now, what kind of harm can it cause?

I keep deleting them and they keep connecting. What's going on? If something is wrong how could I prevent other wallet's connecting to my PC? I added some inbound and outbound rules under Windows Defender. That could be the reason? I also added some peer address found on Reddit, because I was not able to sync. Adding peers manually have any risks?

Thank you.

![chia connections](https://user-images.githubusercontent.com/83424660/116601334-11ae6480-a933-11eb-9ec3-d60b5158511f.jpg)

**Comment by byekity** [comment]
At least someone could confirm if my wallet and funds are safe please?

Some lines with this IP from my log file:

`2021-04-29T21:02:27.675 full_node full_node_server        : INFO     -> new_peak_wallet to peer 109.196.114.26 70067b3d1a2fc2dc008d45a2f99de7b08bb7462722fffac1f3a6d2fbfc1a6ccd

2021-04-29T21:05:48.327 full_node full_node_server        : INFO     <- request_block_header from peer 70067b3d1a2fc2dc008d45a2f99de7b08bb7462722fffac1f3a6d2fbfc1a6ccd 109.196.114.26

2021-04-29T21:05:48.340 full_node full_node_server        : INFO     -> respond_block_header to peer 109.196.114.26 70067b3d1a2fc2dc008d45a2f99de7b08bb7462722fffac1f3a6d2fbfc1a6ccd`

Should these directed to my own IP only, 127.0.0.1?

**Comment by lvcivs** [comment]
I think that just means that some of your peers are running a wallet only instead of a full node. The wallet still needs to be synced, so there's a need to be connected to peers.

So I guess it's nothing to worry about. But maybe others can comment who know more about it.

**Comment by furkan16** [comment]
Hi,
I have the same, is this a security weakness? Or does anyone know what happened?

**Comment by cesargdmi** [comment]
The same thing happens to me, after opening port 8444 on my router strange portfolios of unknown IPs began to appear, and almost always when the chia blockchain appears it loses synchronization, I no longer trust chia, it has too many errors

**Comment by kushariforyou** [comment]
Following to see the explanation
@cesargdmi You can't make assumptions without knowing anything. That's how false information spreads.

**Comment by cesargdmi** [comment]
> Following to see the explanation
> @cesargdmi You can't make assumptions without knowing anything. That's how false information spreads.

Hello.

If you have so much knowledge of the subject of chia, could you please explain to us what those wallets that try to connect to the chia blockchain client mean?

![image](https://user-images.githubusercontent.com/2000465/118890285-090ed580-b8c4-11eb-922c-34e3d8ecdeed.png)


Today it happened to me again, and it is happening to me very often that strange wallets appear, and additionally it is constantly desynchronized with attempted peer connections without height, could you tell us what these connections mean because I have tried to search in the FAQ of chia and in forums but not there is an answer, this makes the project very suspicious.

![image](https://user-images.githubusercontent.com/2000465/118890330-188e1e80-b8c4-11eb-8a94-bff77956bf73.png)

Best regards

**Comment by kushariforyou** [comment]
> > Following to see the explanation
> > @cesargdmi You can't make assumptions without knowing anything. That's how false information spreads.
> 
> Hello.
> 
> If you have so much knowledge of the subject of chia, could you please explain to us what those wallets that try to connect to the chia blockchain client mean?
> 
> ![image](https://user-images.githubusercontent.com/2000465/118890285-090ed580-b8c4-11eb-922c-34e3d8ecdeed.png)
> 
> Today it happened to me again, and it is happening to me very often that strange wallets appear, and additionally it is constantly desynchronized with attempted peer connections without height, could you tell us what these connections mean because I have tried to search in the FAQ of chia and in forums but not there is an answer, this makes the project very suspicious.
> 
> ![image](https://user-images.githubusercontent.com/2000465/118890330-188e1e80-b8c4-11eb-8a94-bff77956bf73.png)
> 
> Best regards

I don't have much knowledge. But it doesn't make the project suspicious. Just because you don't understand something, doesn't make it suspicious. You can go ask the Devs in Keybase. Again I want to be very clear, if you don't understand something, that doesn't make it shady, that just means you don't know what you're talking about.

**Comment by avitron78** [comment]
Yo se un poco del tema por así decirlo, ahí lo que pone es que el tipo de conexión esa lo unico que hace es sincronizarte la wallet y ayudar a sincronizar a otras personas, no se trata de la wallet de otro pavo que te va a secuestrar tus chias XD

**Comment by avitron78** [comment]
Y con respecto a las otras que marcaste que no ponen ninguna altura se trata de ips que estan por así decirlo en tu agenda que se va creando de cuando estableces conexión con alguien, al no darte altura lo que ocurre es que dicho nodo está muy saturado o simplemente esa persona se desconectó, con lo que si esperas se desconectará ese nodo y te buscará otro, es normal ver nodos con distintas alturas, yo lo que suelo hacer es darle a desconectar a aquellas que por así decirlo no me están aportando nada, ya sea no aportando una altura o una altura demasiado baja con respecto al resto de pares. Espero haber aclarado un poco el asunto.

**Comment by Plopsi** [comment]
Normal, while no wallet only client available yet, you can do it and only run a wallet, without a full node. Of course a wallet needs to sync too, while not as critical to keep the block-height as a full-node, espeacially with famers, it still needs to sync to see it's balance.

If you look into the config.yaml in chia you see there are also limits on how many wallets may be connected to a full node - which make sense, as a full node should try to keep up to the chain as close as possible and a wallet don't share blocks because of resonable reasons.

So while unusual (yet) this shouldn't be concernible.

TLDR;
It's fine, as far as I can tell.

**Comment by cesargdmi** [comment]
Hola, gracias a avitron78 y a plopsi por sus explicaciones, quedo mas tranquilo, gracias

Hi, thanks to aviltron 78 and plopsi by you explication, I am more relaxed, thank you
